flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableFactoryTest.java

```diff
@@ -174,6 +175,7 @@ public void testJdbcLookupProperties() {
                         .setCacheMaxSize(1000)
                         .setCacheExpireMs(10_000)
                         .setMaxRetryTimes(10)
+                        .setExcludeEmptyQueryResult(true)
```

Comment:
you need to a new case instead of override the default value

Reply:
you mean new test case,right?, I have added another one,